### ISSUE TITLE
NoSuchMethodError when using together with latest AWS SDK (1.11.461)

### ISSUE THREAD

**Issue by anjo-swe** [aws/aws-lambda-java-libs#64]
**Versions:**
* aws-lambda-java-events: 2.2.4
* aws-lambda-java-core: 1.2.0

**Code:**
```
AWSKMSClientBuilder.defaultClient();
```

**Error:**
```
"errorMessage": "com.amazonaws.client.AwsSyncClientParams.getAdvancedConfig()Lcom/amazonaws/client/builder/AdvancedConfig;",
"errorType": "java.lang.NoSuchMethodError",
"stackTrace": [
  "com.amazonaws.services.kms.AWSKMSClient.<init>(AWSKMSClient.java:454)",
  "com.amazonaws.services.kms.AWSKMSClient.<init>(AWSKMSClient.java:438)",
  "com.amazonaws.services.kms.AWSKMSClientBuilder.build(AWSKMSClientBuilder.java:61)",
  "com.amazonaws.services.kms.AWSKMSClientBuilder.build(AWSKMSClientBuilder.java:27)",
  "com.amazonaws.client.builder.AwsSyncClientBuilder.build(AwsSyncClientBuilder.java:46)",
  "com.amazonaws.services.kms.AWSKMSClientBuilder.defaultClient(AWSKMSClientBuilder.java:45)",
  ...
]
```

**Note:** Current workaround is to stay on `1.11.163` for AWS SDK

**Comment by camhart** [comment]
1.11.288 is the latest version I can find.  This is quite outdated though...  anyone have a more recent one?

**Comment by mergentkm** [comment]
> 1.11.288 is the latest version I can find. This is quite outdated though... anyone have a more recent one?

Were you able to figure this out? I am facing the same issue

**Comment by carlzogh** [comment]
Hey all - `aws-lambda-java-events` version `3.0.0` was just [released](https://search.maven.org/artifact/com.amazonaws/aws-lambda-java-events/3.0.0/jar) with support for all events (including S3) without the inclusion of SDK v1 dependencies.
This should allow you to use any version of the AWS SDK without running into this error as the events library no longer depends on the v1 SDK.

I have also tested @anjo-swe's use case with the latest available version of the KMS v1 SDK [1.11.784](https://search.maven.org/artifact/com.amazonaws/aws-java-sdk-kms/1.11.784/jar) and validated that it now works.
